src/main/java/core/basesyntax/SimpleCalculator.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class SimpleCalculator {
+
+    public double calculate(double a, Character sign, double b) {
```

Comment:
```suggestion
    public double calculate(double a, char sign, double b) {
```
I can see now why you used `equals()` at first
you are right that we should compare objects with `equals`
but we can use primitive type here